multimodal/src/autogluon/multimodal/utils/save.py

```diff
@@ -139,10 +138,10 @@ def setup_save_path(
     elif proposed_save_path is not None:  # TODO: distinguish DDP and existed predictor
         save_path = process_save_path(path=proposed_save_path, raise_if_exist=(raise_if_exist and rank == 0))
     elif old_save_path is not None:
-        if model_loaded or fit_called:
+        if fit_called:
             save_path = process_save_path(path=old_save_path, raise_if_exist=False)
         else:
-            save_path = os.path.abspath(os.path.expanduser(old_save_path))
+            save_path = process_save_path(path=old_save_path, raise_if_exist=(raise_if_exist and rank == 0))
```

Comment:
We may need a multi-GPU machine in the CI to test for multi-GPU training next time.

Reply:
Agree. Maybe select partial tests to avoid slow down CI.